<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="play">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>PLAY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="result">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="ROCK">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>ROCK</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Select Your Move</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PAPER">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>PAPER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SCISSORS">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>211</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>SCISSORS</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="computerChoice_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>computerChoice_label</string>
      </property>
     </widget>
     <widget class="QLabel" name="result_label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string> result_label</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Play_Again">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Play_Again</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="Win_Count">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Wins = 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="Lose_Count">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Loses = 0</string>
      </property>
     </widget>
     <widget class="QLabel" name="Draw_Count">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>251</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Draws = 0</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="Total_Games">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Games = 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>372</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
